src/SDKs/CognitiveServices/Management.CognitiveServices/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
@felixwa what is the usual practice is, you only update AssemblyVersion when you update the major version.
If you have introduced a breaking change, then simply update
AssemblyFileVersion to 2.0.0.0
AssemblyVersion to 2.0.0.0